apps/git-pulse/src/app.ts

```diff
@@ -0,0 +1,17 @@
+import express from "express";
+import routes from "./routes/index"
+const app = express();
+
+// Routing
+app.use('/', routes);
+
+// Error catching endware.
+app.use((err, req, res, next) => {
+  // eslint-disable-line no-unused-vars
+  const status = err.status || 500;
+  const message = err.message || err;
+  console.error(err);
+  res.status(status).send(message);
+});
+
+export default app;
```

Comment:
Please, always add an empty line at the end of each file.
Reasons: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3